integration/run_tests.sh

```diff
@@ -131,8 +131,8 @@ assert_ExitCodeForCommand "6" "${bin_path} -playbook ${root_key}/good-postgres-w
 assert_ExitCodeForCommand "6" "${bin_path} -playbook ${root_key}/good-postgres-with-template.yml"
 assert_ExitCodeForCommand "0" "${bin_path} -playbook ${root_key}/good-postgres-with-template.yml -var username=postgres,password=,host=localhost"
 
-# Test: Truncated steps field in playbook should return exit code 8
-assert_ExitCodeForCommand "8" "${bin_path} -playbook ${root_key}/good-postgres-truncated.yml -lock ${root}/dist/integration-lock"
+# Test: Truncated steps field in playbook should return exit code 1
```

Comment:
Not that it's wrong but any particular reason to change from 8 to 1 here?

Reply:
The reason is that now invalid/truncated playbooks fail from [this line](https://github.com/snowplow/sql-runner/blob/8279f9654f315de25ab4b7c1598e583ee18a62bf/sql_runner/main.go#L58). The thought was to match the other exit scenarios from main (through `log.Fatal`) that happen before `Run`. Also, exit code `8` is tied to "no queries to run", which may not exactly reflect "no targets" or "no steps". Not a strong opinion on this though, happy to change!